relay-general/src/store/transactions.rs

```diff
@@ -373,10 +370,10 @@ mod tests {
     }
 
     #[test]
-    fn test_discards_on_missing_op_in_context() {
+    fn test_defaults_missing_op_in_context() {
         let mut event = Annotated::new(Event {
             ty: Annotated::new(EventType::Transaction),
-            timestamp: Annotated::new(Utc.ymd(2000, 1, 1).and_hms(0, 0, 0)),
+            timestamp: Annotated::new(Utc.ymd(2000, 1, 1).and_hms(0, 0, 10)),
```

Comment:
Changed this such that the transaction now has non-zero duration.
Zero duration is not the common case, and we don't want to mix up intentions in the test.
This test is about missing `op` in `event.contexts.trace` => `event.contexts.trace = "op"` (default value).